api/src/middlewares/services/utils/additionalContexts.json

```diff
@@ -7,7 +7,8 @@
       "localRequirements": "",
       "civilPartnership": false,
       "postAddress": "",
-      "duration": "3 months"
+      "duration": "3 months",
+      "postal": "FALSE"
```

Comment:
is it easy enough to cast this into an actual boolean? or have a util function that checks for "FALSE"?